public/app/ui/components/display/typed-list.html

```diff
@@ -2,7 +2,7 @@
     <!-- ko if: hasValue -->
     <ul>
         <!-- ko foreach: { data: value, as: 'item' } -->
-        <li class="text-nowrap">
+        <li>
```

Comment:
Not sure why this was added but it was causing overlap with the image when the value was long (e.g. Creator).